src/engine/Input/Pointer.ts

```diff
@@ -236,7 +240,7 @@ export class Pointers extends Class {
          var x: number = e.pageX - Util.getPosition(this._engine.canvas).x;
          var y: number = e.pageY - Util.getPosition(this._engine.canvas).y;
          var transformedPoint = this._engine.screenToWorldCoordinates(new Vector(x, y));
-         var pe = new PointerEvent(transformedPoint.x, transformedPoint.y, 0, PointerType.Mouse, e.button, e);
+         var pe = new PointerEvent(transformedPoint.x, transformedPoint.y, e.pageX, e.pageY, 0, PointerType.Mouse, e.button, e);
```

Comment:
Is pageX and pageY the right value to use? What if the canvas for the game is centered in the middle of the screen? Don't we want the screen coordinates where top left of the canvas element is (0, 0) regardless of where the canvas is on the screen?

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/pageX